tools/submission/submission_checker.py

```diff
@@ -1527,7 +1544,7 @@ def __init__(
         self.seeds = self.base["seeds"]
         self.test05_seeds = self.base["test05_seeds"]
         self.accuracy_target = self.base["accuracy-target"]
-        self.accuracy_upper_limit = self.base["accuracy-upper-limit"]
+        self.accuracy_upper_limit = self.base.get("accuracy-upper-limit")
```

Comment:
Let's set an empty default value to avoid errors
```python3
self.accuracy_upper_limit = self.base.get("accuracy-upper-limit", {})
```

Reply:
Thank you Pablo. I missed that it's a dictionary. Fixed now.